control/veda_heal.py

```diff
@@ -65,8 +65,8 @@ def discovery(self):
                 hours=self.auth_dict['heal_end']
             )
         )
-
-        self.send_encodes()
+        if self.auth_dict['rabbitmq_broker'] is not None:
+            self.send_encodes()
```

Comment:
Why do we need this check ? is it for tests ? If yes then please mock the method for tests.